js/rpg_core/ImageCache.js

```diff
@@ -0,0 +1,124 @@
+function ImageCache(){
+    this.prototype.initialize.call(this);
```

Comment:
I knew that `this.initialize.apply` was used in constructor functions (for delegate to instance methods), `ClassName.prototype.initialize.call` were used otherwise (for call super methods). Let's do the same.